### PR TITLE
makes nockOpts nullable

### PR DESCRIPTION
**Pull request by big-kahuna-burger**
Defaults fixtures dir to a ```/fixtures``` sibling location relative to currently executing script

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -1,9 +1,7 @@
-var path = require('path')
+//
 var tape = require('tape')
 
-var test = require('../')(tape, {
-  fixtures: path.join(__dirname, 'fixtures')
-})
+var test = require('../')(tape)
```
Comment (Flet) [inline test/index.js:4]: I'd prefer two have two tests: one with no options and one with options that have "fixtures" in a different location.

File: index.js [hunk 2]
```diff
@@ -2,11 +2,12 @@
...
 var nock = require('nock')
 var sanitizeFilename = require('sanitize-filename')
+var path = require('path')
 
 function tapeNockFactory (tapeTest, nockOpts) {
   var nockBack = nock.back
-
-  nockBack.fixtures = nockOpts.fixtures
+  nockOpts = nockOpts || {}
+  nockBack.fixtures = nockOpts.fixtures || path.join(path.dirname(module.parent.filename), 'fixtures')
```
Comment (Flet) [inline index.js:10]: 👍

**Review by Flet — APPROVED** [review]

**Comment by Flet** [comment]
squash/merging this now! We can tinker with tests afterwards.

**Comment by Flet** [comment]
Eh, I suppose the "only" tests cover an explicit fixtures path. Good enough for me.

**Comment by Flet** [comment]
Published as `tape-nock@1.5.0`

Thanks for contributing! What is your npm username @big-kahuna-burger? I'll add you as a someone who can `npm publish` if you wish.

**Comment by big-kahuna-burger** [comment]
It's same as here, thanks for adding.

https://www.npmjs.com/~big-kahuna-burger

**Comment by big-kahuna-burger** [comment]
I'll see later on to add coveralls so we can have coverage report on this repo. It's neat and small code base.

**Comment by Flet** [comment]
@big-kahuna-burger thank you for volunteering your time to open source :)

**Pull request merged**